react/CodeBlock.js

```diff
@@ -0,0 +1,81 @@
+const React = require('react')
+const ReactDOM = require('react-dom')
+const { Prism } = require('react-syntax-highlighter')
+const { dark } = require('react-syntax-highlighter/dist/cjs/styles/prism')
+const { Tab, Tabs, TabList, TabPanel } = require('react-tabs')
```

Comment:
Can we use the Primer component here instead? https://primer.style/components/TabNav

Reply:
Same answer as https://github.com/github/docs/pull/610#discussion_r508502786